contrib/swagger.yaml

```diff
@@ -0,0 +1,439 @@
+swagger: '2.0'
+info:
+  title: XSnippet API
+  description: REST API of XSnippet, a simple web-service for sharing code snippets
+  version: "4.0.0"
+
+host: api.xsnippet.org
+
+schemes:
+  - http
+  - https
+
+consumes:
+  - application/json
+
+produces:
+  - application/json
+
+paths:
+  /snippets:
+    get:
+      summary: Retrieve information about code snippets.
+      description: |
+        The Snippets endpoint returns information about stored code snippets. The
+        listed snippets are sorted by the date-time of creation in the ascending order.
+
+        By default, only `limit` latest snippets are returned. Paginated queries
+        should be used in order to get more snippets.
+      parameters:
+        - name: filetype
```

Comment:
I'm ok with `filetype`, however, maybe it makes sense to sense to rename it into `syntax`? What do you think?